spec/Section 5 -- Validation.md

```diff
@@ -1636,13 +1734,14 @@ query takesCatOrDog($catOrDog: CatOrDog) {
     transitively.
   - For each {fragment} in {fragments}:
     - For each {variableUsage} in scope of {fragment}, variable must be in
-      {operation}'s variable list.
+      {fragment}'s or {operation}'s variable list.
```

Comment:
Might be worth making it clear this is a "logical or" (allowing both to be true) rather than an "exclusive or" (XOR) which is more commonly how a layman interprets the word `or`.